packages/shared/providers.yaml

```diff
@@ -3930,6 +3930,32 @@ okta:
             example: domain
             suffix: .okta.com
 
+okta-preview:
+    display_name: Okta (Preview)
+    auth_mode: OAUTH2
+    authorization_url: https://${connectionConfig.subdomain}.oktapreview.com/oauth2/v1/authorize
+    token_url: https://${connectionConfig.subdomain}.oktapreview.com/oauth2/v1/token
+    authorization_params:
+        response_type: code
+        response_mode: query
+    token_params:
+        grant_type: authorization_code
+    refresh_params:
+        grant_type: refresh_token
+    proxy:
+        base_url: https://${connectionConfig.subdomain}.oktapreview.com
+        retry:
+            at: 'x-rate-limit-reset'
+    docs: https://docs.nango.dev/integrations/all/okta
+    connection_config:
+        subdomain:
+            type: string
+            title: Okta Preview Domain
+            description: The subdomain of your Okta Preview account
+            pattern: '^[a-z0-9_-]+$'
+            example: domain
+            suffix: .oktapreview.com
+
```

Comment:
cannot it not be an alias of okta just overriding the urls?

Reply:
Yup https://github.com/NangoHQ/nango/pull/2960/commits/25d01d3a9b91afc813b7d8b8b72497a1395da419